### PR TITLE
Follow-up edits to PR#2647

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/2647

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @bfallonf @tpoitras PTAL

File: admin_guide/image_policy.adoc [hunk 1]
```diff
@@ -10,24 +10,23 @@
...
+admission plug-in (currently considered beta). It allows you to control:
 
- 1. the source of images: which registries can be used to pull images
- 2. image resolution: force pods to run with immutable digests to ensure the image does not change due to a re-tag
- 3. docker image label restrictions: force an image to have or not have particular labels
- 4. image annotation restrictions: force an image in the integration docker registry to have or not have particular annotations
+. *The source of images*: which registries can be used to pull images
+. *Image resolution*: force pods to run with immutable digests to ensure the image does not change due to a re-tag
+. *Docker image label restrictions*: force an image to have or not have particular labels
+. *Image annotation restrictions*: force an image in the integration docker registry to have or not have particular annotations
```
Comment (bfallonf) [inline admin_guide/image_policy.adoc]: @ahardin-rh I think the above should be bullet points rather than a list.

**Comment by bfallonf** [comment]
@ahardin-rh Just one comment. LGTM otherwise!

**Comment by ahardin-rh** [comment]
@bfallonf Thanks for the review!

**Comment by ahardin-rh** [comment]
rev history in original PR

**Pull request merged**